app/controllers/portal/tax_returns_controller.rb

```diff
@@ -42,9 +42,8 @@ def success; end
     private
 
     def load_client_tax_return
-      @tax_return = current_client.tax_returns.includes(client: [:intake]).find(params[:tax_return_id])
-    rescue ActiveRecord::RecordNotFound
-      redirect_to :root
+      @tax_return = current_client.tax_returns.includes(client: [:intake]).find_by(id: params[:tax_return_id])
```

Comment:
This was an update requested by @alaroia-cfa on the previous PR. Instead of capturing the error raised by a #find, we use find_by to return nil, and then redirect if the value is nil.